packages/core/src/domain/error/computeStackTrace.ts

```diff
@@ -1,11 +1,34 @@
+/**
+ *
+ * Cross-browser stack trace computation.
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
/**
 * Cross-browser stack trace computation.
```